### PR TITLE
add support for role

### PR DESCRIPTION
**Pull request by vieux**
Signed-off-by: Victor Vieux <victorvieux@gmail.com>

### CODE REVIEW

File: cli/help.go [hunk 1]
```diff
@@ -47,6 +47,7 @@ Options:
                                     {{printf "\t * mesos.address=\taddress to bind on [$SWARM_MESOS_ADDRESS]"}}
                                     {{printf "\t * mesos.port=\tport to bind on [$SWARM_MESOS_PORT]"}}
                                     {{printf "\t * mesos.offertimeout=10m\ttimeout for offers [$SWARM_MESOS_OFFER_TIMEOUT]"}}
+                                    {{printf "\t * mesos.role=\trole for offers [$SWARM_MESOS_ROLE]"}}
```
Comment (aluzzardi) [inline cli/help.go]: What's the default? And what are the possible options? We should reject invalid options (e.g. `mesos.role=drunk`)

**Comment by aluzzardi** [comment]
Can you add a more explicit commit message?

**Comment by aluzzardi** [comment]
Is this `changelog` worthy? I added the `impact` label

File: cli/help.go (hunk 1 above)
Comment (vieux) [inline cli/help.go]: any string is valid, it's defined by the mesos master, so yes `drunk` can be valid.

Comment (vieux) [inline cli/help.go]: :)

**Comment by vieux** [comment]
closing for now, I need to add test before merging

**Pull request closed without merging**